### PR TITLE
[Docs]: add section to docs about how to update docs

### PR DESCRIPTION
**Pull request by MagdalenaHenke**
Documenting the outcome of this [Clubhouse](https://app.clubhouse.io/blocks/story/44379/determine-review-process-for-documentation)

Basically putting [this doc](https://docs.google.com/document/d/1LLZg1Z7bzF4TzO0XN3y4ArWwLVC8f6p3qst7_9VMbC0/edit#) on the docs page.

The new page:
Screen Shot 2019-11-21 at 3 53 58 PM

### CODE REVIEW

File: packages/blocks-docs/src/css/page.styl [hunk 1]
```diff
@@ -109,34 +109,34 @@ div[role="group"][tabindex] {
   }
 }
 
-.writing-guidelines-list {
+.help-text-list {
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/css/page.styl]: reusing this stying on another page, so I made the class name more generic

File: packages/blocks-docs/src/pages/help/documentation-updates.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React from "react"
+
+import Layout from "../../components/layout"
+import Header from "../../components/header"
+
+const WritingGuidelinesPage = () => (
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/pages/help/documentation-updates.js]: Is this copy ok? I'm not a copy person and didn't spend lots and lots of time thinking exactly about what this copy should be (I thought about the content, but not as much the voice).

**Comment by MagdalenaHenke** [comment]
@judyhe - another one that doesn't need eng review, but I feel iffy putting copy on the docs that _no one_ has approved. Should you be the person to do that? Can I just go for it? Is Laura?

**Comment by judyhe** [comment]
👍

**Pull request merged**